chart/tests/README.md

```diff
@@ -0,0 +1,25 @@
+
+### local dev testing instructions
+
+```bash
+
+# change automated parts of templates
+test_image="rancher/rancher"
+test_image_tag="v2.7.0"
+sed -i -e "s/%VERSION%/${test_image_tag}/g" ./chart/Chart.yaml
+sed -i -e "s/%APP_VERSION%/${test_image_tag}/g" ./chart/Chart.yaml
+sed -i -e "s@%POST_DELETE_IMAGE_NAME%@${test_image}@g" ./chart/values.yaml
+sed -i -e "s/%POST_DELETE_IMAGE_TAG%/${test_image_tag}/g" ./chart/values.yaml
+
+# install plugin if necessary
+helm plugin install --version {version}  https://github.com/rancher/helm-unittest
```

Comment:
This didn't work for me, I had to do

```
export ARCH=amd64
export CATTLE_HELM_UNITTEST_VERSION=v0.1.7-rancher4
helm plugin install  https://github.com/rancher/helm-unittest
```

Reply:
Looks like that works on a linux box because of how the script is setup: https://github.com/rancher/helm-unittest/blob/master/install-binary.sh#L10

No support for mac in there, but the darwin binary does exist on the release: https://github.com/rancher/helm-unittest/releases/

I'll add both options to readme.